.github/workflows/main.yml

```diff
@@ -32,13 +35,29 @@ jobs:
           ${{ matrix.platform.msys-env }}-cmake
           ${{ matrix.platform.msys-env }}-gcc
           ${{ matrix.platform.msys-env }}-libjpeg-turbo
+          ${{ matrix.platform.msys-env }}-libjxl
```

Comment:
You can add `libavif` here, too.

Reply:
We want to use the versions we ship with SDL_image, right?

Reply:
Autofoo doesn't do that

Reply:
You can switch between vendored and system packages using the `SDL2IMAGE_VENDORED` option.
There exist package managers on all platforms that would make us vendoring the 3rd party libraries not necessary anymore.
It would also make the cmake script much cleaner.

Reply:
> You can switch between vendored and system packages using the `SDL2IMAGE_VENDORED` option.

Cmake: yes. Autotools doesn't do that , as far as I know.

Reply:
> You can add `libavif` here, too.

We need libavif 0.10
https://github.com/madebr/SDL_image/runs/6479514233?check_suite_focus=true#step:7:25

Reply:
> We need libavif 0.10

Do we really, though? 0.10 is what we forked from mainstream, can't we
run against something somewhat older? (@slouken?)

Reply:
The error is very benign, 0.10 added `strictFlags`:
```
/home/maarten/programming/SDL_image/IMG_avif.c:257:12: error: ‘avifDecoder’ has no member named ‘strictFlags’
  257 |     decoder->strictFlags = AVIF_STRICT_DISABLED;
      |            ^~
/home/maarten/programming/SDL_image/IMG_avif.c:257:28: error: ‘AVIF_STRICT_DISABLED’ undeclared (first use in this function)
  257 |     decoder->strictFlags = AVIF_STRICT_DISABLED;
```

Reply:
Then how did the autotools build succeed in that CI run?

Reply:
According to git blame, strict flags was in v0.9.1:
https://github.com/AOMediaCodec/libavif/commit/9b942c24d3c3cb0547d79e253d013da7542edad0

Reply:
Thanks, I will update the minimum version then.
I tested on a Linux Distribution with 0.9.0 and did not look up the minimum libavif version required.

Reply:
I can't add libavif-dev for Ubuntu, as it's for Ubuntu 21.10+ only: https://packages.ubuntu.com/search?keywords=libavif-dev

Reply:
> I can't add libavif-dev for Linux, as it's for Ubuntu 21.10+ only:

That's OK, we'll have something to test-build with mingw at least.

Reply:
`${{ matrix.platform.msys-env }}-libavif` has been added.